pkg/config/config.go

```diff
@@ -492,6 +492,10 @@ type Update struct {
 	VersionTransform []VersionTransform `json:"version-transform,omitempty" yaml:"version-transform,omitempty"`
 	// ExcludeReason is required if enabled=false, to explain why updates are disabled.
 	ExcludeReason string `json:"exclude-reason,omitempty" yaml:"exclude-reason,omitempty"`
+	// Polling indicates that polling should be used to find latest versions
+	Polling *Polling `json:"polling,omitempty" yaml:"polling"`
```

Comment:
Just curious, should this have `omitempty` for YAML too?

Reply:
yes! nice catch :)